apps/examples/lcd_test/example_lcd.c

```diff
@@ -321,66 +322,76 @@ static void test_quad(void)
 	free(lcd_data);
 }
 
+static void release_frame_buffer(struct lcddev_area_s *area)
+{
+	if (area->data != NULL) {
```

Comment:
I think she does not mean checking `area->data` with null.
I think, it's `area`. If it's null, then `free(area->data);` will be null dereferencing.

But if there is no chance to be null for area, no need to check it.